doc/api/stream.md

```diff
@@ -1159,6 +1159,31 @@ readable stream will release any internal resources.
 Implementors should not override this method, but instead implement
 [`readable._destroy`][readable-_destroy].
 
+##### readable[Symbol.asyncIterator]
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Returns an [AsyncIterator][async-iterator] to fully consume the stream.
```

Comment:
`[AsyncIterator][async-iterator]` -> `[AsyncIterator][stream-iterator]` ? Or the bottom reference can be edited.

Reply:
fixed in the bottom reference.